sim/battle.ts

```diff
@@ -3164,18 +3163,26 @@ export class Battle extends Dex.ModdedDex {
 				winner: this.winner,
 				seed: this.prngSeed,
 				turns: this.turn,
-				p1: this.p1.name,
-				p2: this.p2.name,
-				p1team: this.p1.team,
-				p2team: this.p2.team,
-				score: [this.p1.pokemonLeft, this.p2.pokemonLeft],
+				p1: this.sides[0].name,
+				p2: this.sides[1].name,
+				p3: this.sides[2] && this.sides[2].name,
```

Comment:
I would pull these out so they don't appear in two player games.

Reply:
```
if (!log.p3) delete log.p3;
if (!log.p4) delete log.p4;
if (!log.p3team) delete log.p3team;
if (!log.p4team) delete log.p4team;
```
?

Reply:
or 
```
if (sides[2] && sides[3]) {
   log.p3 = this.sides[2].name;
   log.p4 = this.sides[3].name;
   log.p3team = this.sides[2].team;
   log.p4team = this.sides[3].team;
}
```

Reply:
Either works. I prefer the latter approach. Also append everything else after, so it's still in order.

Reply:
If order matters, then the former makes way more sense to me.